kore/src/Kore/Attribute/Location.hs

```diff
@@ -0,0 +1,93 @@
+{-|
+Module      : Kore.Attribute.Location
+Description : Line/column location attribute
+Copyright   : (c) Runtime Verification, 2019
+License     : NCSA
+Maintainer  : vladimir.ciobanu@runtimeverification.com
+
+-}
+module Kore.Attribute.Location
+    ( Location (..)
+    , LineColumn (..)
+    ) where
+
+import           Control.Applicative
+                 ( optional )
+import           Control.DeepSeq
+                 ( NFData )
+import           Data.Default
+import qualified Data.Text as Text
+import           GHC.Generics
+                 ( Generic )
+import           Text.Megaparsec
+                 ( Parsec, parseMaybe )
+import           Text.Megaparsec.Char
+import           Text.Megaparsec.Char.Lexer
+                 ( decimal )
+
+import           Kore.AST.Kore
+import           Kore.Attribute.Parser
+                 ( ParseAttributes (..) )
+import qualified Kore.Attribute.Parser as Parser
+
+data LineColumn = LineColumn
+    { line   :: Int
+    , column :: Int
+    } deriving (Eq, Ord, Show, Generic)
+
+instance NFData LineColumn
+
+instance Default LineColumn where
```

Comment:
I'm not sure this `Default` instance makes sense. It doesn't seem to be used, is that right?